suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -0,0 +1,50 @@
+import { describe, test, beforeAll, expect } from '@jest/globals'
+import { newCeramic } from '../../utils/ceramicHelpers.js'
+import { createDid } from '../../utils/didHelper.js'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { Model } from '@ceramicnetwork/stream-model'
+import { ModelInstanceDocument } from '@ceramicnetwork/stream-model-instance'
+import { newModel, basicModelDocumentContent } from '../../models/modelConstants'
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { CommonTestUtils as TestUtils } from '@ceramicnetwork/common-test-utils'
+import { utilities } from '../../utils/common.js'
+
+const delay = utilities.delay
+const ComposeDbUrls = String(process.env.COMPOSEDB_URLS).split(',')
+const adminSeeds = String(process.env.COMPOSEDB_ADMIN_DID_SEEDS).split(',')
+const nodeSyncWaitTimeSec = 2
+
+describe('Model Integration Test', () => {
+  let ceramicNode1: CeramicClient
+  let ceramicNode2: CeramicClient
+  let modelId: StreamID
+  beforeAll(async () => {
+    const did1 = await createDid(adminSeeds[0])
+    const did2 = adminSeeds[1] ? await createDid(adminSeeds[1]) : did1
```

Comment:
Continuing the conversation on matching up DID with URLs. Should we error instead of defaulting to a did from another node? The expectation is that the env vars supply a DID for every URL. If we do not have one I think the test setup should just error so its clear what is wrong. Otherwise we might get opaque errors from tests about a DID being invalid.

Reply:
Reason for adding this in the first place : 
So when running this on local correct me if I am wrong, we might only get one which works for both the urls. While testing this locally adminSeeds[0] works for both composedburl[0] and composedburl[1]. 

This is not true on durable infra. 

On durable infra this code explicitly expects there to be adminSeeds[0] which corresponds to composedburl[0] and adminSeeds[1] for composedburl[1] exactly. Anything else that is showing up is wrong according to and is not verified by the test. I would much rather have a 2D list of adminDids for each url. 

Why its not ideal and other things I can do : 
This is a bit hacky, the other thing I can do is check if we are running the tests locally or on durable infra which I'm not sure is worth it.

Reply:
Or we can just error out saying that each composedburl should have a did be supplied be it local or durable env. And the developer has to set go worry about that if the tests fail on local.